libs/amplify/.dart_tool/package_config_subset

```diff
@@ -0,0 +1,173 @@
+amplify
+2.16
+file:///C:/University/COS301/FINAL%20CLONE/ReverseHand/libs/amplify/
```

Comment:
Like this is build based on your file system... it won't work if pulled down by another team member or on the build fleet?